tests/languages/asmatmel/comment_feature.test

```diff
@@ -0,0 +1,13 @@
+;
+; foo bar baz
+
+-------------------------
+
+[
+  ["comment", ";"],
```

Comment:
Tabs for indentation also applies to tests.

Reply:
Fixed.